src/components/organisms/Accordion/AccordionHeader/AccordionHeader.tsx

```diff
@@ -76,9 +72,9 @@ const AccordionHeader = ({ children, id, index, textSize = 'body', onClick, ...r
   return (
     <StyledButton type="button" ref={ref} onClick={handleClick} {...headerPropsRest} {...rest}>
       <TitleContainer>
-        <Title weight="bold" size={textSize}>
+        <Heading as="h3" className="my-0" color={colors.greyDarkest} size="h6">
           {children}
-        </Title>
+        </Heading>
         <Cross active={isActiveSection(index)} />
       </TitleContainer>
     </StyledButton>
```

Comment:
<img width="1090" alt="Screenshot 2022-12-12 at 08 56 38" src="https://user-images.githubusercontent.com/3974301/207003868-592a0606-08fe-4acb-abc0-28aa6646de80.png">

Worth mentioning that adding a heading inside a button is semantically incorrect

Reply:
@moshie you are totally right! Girls, we have done it wrong, it should be <h3><button /> </h3> 
![image](https://user-images.githubusercontent.com/27281884/207007493-fc6c7005-a8a5-4336-9921-c1c7cd1bccf8.png)

Reply:
i've made changes to put the button inside the heading now